airgun/widgets.py

```diff
@@ -132,6 +132,65 @@ def fill(self, name):
         return self.select(name)
 
 
+class FormRadioGroup(RadioGroup):
```

Comment:
first, we do need to support bad design from UI perspective, so it is preferable to fix it on application side
second, we still need to find solution without JS injections
third, that is not common widget, so it will be good enough to create `class` that will be simple `View` that consists of that two specific radio buttons (in case we need workaround)

Reply:
Fix PR submitted https://github.com/theforeman/foreman/pull/5953
will make a workaround that should affect tests when airgun is updated (when code will be available downstream)